BB-SC/AppLayer/Communication/BB_AOSP_Push_Notification_Service.md

```diff
@@ -0,0 +1,91 @@
+
+# AOSP Push Notification Service
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Communication
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+AppLayer
+
+## Known Implementation
+
```

Comment:
e.g. https://novu.co, likely more

Reply:
Agreed. Will be taken care of in the next pull request.